orca_detector/Dockerfile.dev

```diff
@@ -25,17 +25,42 @@ RUN pip install --upgrade setuptools
 RUN pip3 install jupyter -U && \
     pip3 install jupyterlab
 
+# Clone the repo
+RUN git clone https://github.com/paloukari/OrcaDetector
+
 # Install all dependency libs for the project.
-COPY requirements.pip /
-RUN pip3 install -r ./requirements.pip
+RUN pip3 install -r OrcaDetector/orca_detector/requirements.pip
```

Comment:
The removal of the COPY command allowed me to build the container from a zero touch (x)100 IBM server. 
This will also help us reduce the setup friction for other people who want to dev/run our work.
To your point, my workflow is: create the container, jump in to it and checkout to my branch, install the requirements again and start coding. 
As long as you don't close the container, the frozen layer shouldn't mater.
On the other hand, I think it'd be better move the requirements installation the after the data download, since this way you'll have to rebuild only the last layer in every approved PR.

Reply:
Same as above,I forgot that I was doing a clone on the host too. I'll change this back to COPY

Reply:
reverted back to COPY